Source/tess.c

```diff
@@ -1018,6 +1018,7 @@ int tessTesselate( TESStesselator *tess, int windingRule, int elementType,
 		tess->vertexIndices = 0;
 	}
 
+	TESSindex inputVertexCount = tess->vertexIndexCounter;
```

Comment:
Leave note here, why we need to take a copy. I guess the counter is repurposed, but I cannot remember how.

Reply:
It's reset to 0 just below, so I assume it's reused to index the new vertices in the tessellated mesh.

Reply:
I added a comment to clarify.